legate/tester/stages/util.py

```diff
@@ -27,11 +27,8 @@
 
 UNPIN_ENV = {"REALM_SYNTHETIC_CORE_MAP": ""}
 
-CUNUMERIC_TEST_ENV = {"CUNUMERIC_TEST": "1"}
-
 # Raise min chunk sizes for deferred codepaths to force eager execution
 EAGER_ENV = {
-    "CUNUMERIC_TEST": "0",
     "CUNUMERIC_MIN_CPU_CHUNK": "2000000000",
```

Comment:
Now the NumPy fallback is controlled by `LEGATE_TEST`. Because we set `LEGATE_TEST=1` unconditionally on all tests, the eager stage will now always disable the fallback, even through we're trying to set the min_chunk sizes high enough so we can force-enable it.

Therefore, I think we want to override `CUNUMERIC_DISABLE_FALLBACK=0` in this eager env (once that is an option, as suggested in https://github.com/nv-legate/cunumeric/pull/922).

Reply:
This honestly seems worse than just keeping the status quo (no reduction in env vars, current usage stops working)

Reply:
Yes, but only for the special "eager" case. The more other, more typical testing modes (cpu, gpu, openmp) now have one less variable.

Reply:
https://github.com/nv-legate/legate.core/pull/715/commits/20b4e7c40d165fc704ef34c47bc092f51c17ce68 for update in cn PR